frontend-ssr/package-lock.json

```diff
@@ -0,0 +1,8461 @@
+{
```

Comment:
I think you ran `npm install` here. 

We're using `yarn` so there should be no `package-lock.json` file.